services/ui-src/package.json

```diff
@@ -20,7 +20,7 @@
     "@emotion/styled": "^11.11.5",
     "@hookform/resolvers": "^2.9.11",
     "@vitejs/plugin-react": "^4.3.0",
-    "aws-amplify": "^5.3.19",
+    "aws-amplify": "5.3.4",
```

Comment:
why are we locking this to a past version in ui-src?

Reply:
well i accidentally left the carat off so i added back in, if that was what you were asking 😀